cmd/singularity/cli/capability.go

```diff
@@ -43,13 +44,16 @@ func init() {
 
 // CapabilityCmd is the capability command
 var CapabilityCmd = &cobra.Command{
-	Run:                   nil,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return errors.New("Invalid command")
+	},
 	DisableFlagsInUseLine: true,
 
-	Use:     docs.CapabilityUse,
-	Short:   docs.CapabilityShort,
-	Long:    docs.CapabilityLong,
-	Example: docs.CapabilityExample,
+	Use:           docs.CapabilityUse,
+	Short:         docs.CapabilityShort,
+	Long:          docs.CapabilityLong,
+	Example:       docs.CapabilityExample,
+	SilenceErrors: true,
```

Comment:
What is the purpose of SilenceErrors here? Should this be instead SilenceUsage?

Reply:
I wasn't sure what the desired outcome was. I assumed the expected usage was to return 1 but still show usage, so I silenced the errors. I can swap that around if you prefer the other way.

Also, is it correct (as I've implemented it) to return an error on a blank "singularity" command?